models/sales_and_procurement/star_schema/intermediate/int_sap__company.sql

```diff
@@ -0,0 +1,86 @@
+{{ config(enabled=var('sap_using_t001w', True)) }}
```

Comment:
Isn't this supposed to be referencing `t001`? `t001w` is plants I think. 
```suggestion
{{ config(enabled=var('sap_using_t001', True)) }}
```